.github/hooks/commit-msg

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Modified from https://github.com/chaitanyagupta/gitutils
+
+red='\033[0;31m'
+no_color='\033[0m'
+grey='\033[0;90m'
+
+echo -e "Checking commit message for bad references ${grey}(commit-msg hook)${no_color}"
+
+echo "Message:"
+echo `cat $1`
```

Comment:
Would we be able to nuke these two lines to keep it clean? 😀

Reply:
oh for sure - I just chucked this together as an idea!